sdk/monitor/Microsoft.OpenTelemetry.Exporter.AzureMonitor/src/NDJsonWriter.cs

```diff
@@ -33,6 +33,13 @@ public Memory<byte> ToBytes()
             return Stream.ToArray();
         }
 
+        public override string ToString()
+        {
+            Stream.Position = 0;
+            using var streamReader = new StreamReader(Stream);
+            return streamReader.ReadToEnd();
+        }
```

Comment:
Hey, does resetting the Stream position here affect the other methods?

Sorry if this is a stupid question, honestly don't know so asking. :)

Reply:
Resetting the stream is an issue. In this code `content.ToString()` is the last call on the stream, so resetting is not an issue here.

Reply:
Without setting the `Stream.Position` it won't allow to covert the stream to string.